pennylane/transforms/decompositions/single_qubit_unitary.py

```diff
@@ -14,13 +14,13 @@
 """Contains transforms and helpers functions for decomposing arbitrary unitary
 operations into elementary gates.
 """
-
+import numpy as np
 import pennylane as qml
 from pennylane import math
 
 
 def _convert_to_su2(U):
-    r"""Convert a 2x2 unitary matrix to :math:`SU(2)`.
+    r"""Check unitarity of a matrix and convert it to :math:`SU(2)` if possible.
```

Comment:
It is odd, this seems to be reverting the changes from https://github.com/PennyLaneAI/pennylane/pull/1894, although that commit is already in v0.22.0-rc0 and this PR. :thinking: